xtask/src/main.rs

```diff
@@ -0,0 +1,85 @@
+mod bench;
+mod cli;
+mod nomt;
+mod sov_db;
+mod timer;
+
+use anyhow::Result;
+use bench::bench;
+use clap::Parser;
+use cli::{Backend, Cli, Commands, Workload};
+use nomt::NomtDB;
+use sov_db::SovDB;
+use std::rc::Rc;
+
+pub trait DB {
```

Comment:
Nit: Just to let you know, that according to the Rust naming convention `Db` would've been a better choice.